src/main/java/iabudiab/maven/plugins/dependencytrack/UploadBomMojo.java

```diff
@@ -72,6 +72,12 @@ public class UploadBomMojo extends AbstractDependencyTrackMojo {
 	@Parameter(defaultValue = "60", property = "tokenPollingDuration", required = true)
 	private Integer tokenPollingDuration;
 
+	@Parameter(defaultValue = "3", property = "projectMetricsRetryLimit", required = true)
+	private Integer projectMetricsRetryLimit;
+
+	@Parameter(defaultValue = "5", property = "projectMetricsRetryDelay", required = true)
+	private Integer projectMetricsRetryDelay;
+
```

Comment:
How about using one parameter `projectMetricsPollingDuration` with a fixed delay of 5 seconds. This way it would be analogous to the `tokenPollingDuration`. An alternative is to make the delay in both of those actions configurable.

Reply:
That is a good point. We should keep it consistently. I like to be more flexible and make everything configurable to meet all situations. We can keep the configuration simple, be choosing suitable defaults for most of the cases but we are keeping it flexible enough to overcome situations where these defaults are not the best fit.

So yes, in my opinion we should add the delay configuration for the `tokenPolling` process.

Reply:
Ok, I'll refactor that after this PR